test/sdk_test.rb

```diff
@@ -14,4 +17,70 @@ def test_no_api_key_raises_on_api_call
       sdk.users.find("x")
     end
   end
+
+  def test_verify_token
```

Comment:
@gkats Added some tests, please take a look!

Reply:
Tests look good, thanks for adding them! The indentation seems a bit off though. Is it just Github or the code is indeed improperly formatted?

Reply:
Nope it was me, sorry about that. Should be fine now. I've also re-structured the tests a bit so that they're easier to read, and added some more test cases.